Configurations/unix-Makefile.tmpl

```diff
@@ -1381,7 +1381,7 @@ test_ordinals:
 
 tags TAGS: FORCE
 	rm -f TAGS tags
-	-util/ctags.sh
+	-( cd $(SRCDIR); util/ctags.sh )
```

Comment:
After we go into the `$(SRCDIR)` will `util/ctags.sh` be present? (i.e. is it copied over)

Reply:
The `util/ctags.sh` is actually in $(SRCDIR) not in the build dir.

Reply:
Yes, it's part of the source distro which is based from SRCDIR.